styles.css

```diff
@@ -1,11 +1,10 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400&family=Roboto:wght@300;400;500&display=swap');
 
 body {
-  width: 310px;
-  padding: 35px;
   background-color: #e5e5e5;
   font-family: "poppins", sans-serif;
   font-style: normal;
+  margin: 0;
 }
 
```

Comment:
- Please try to remove the horinsontal scroll in your webpage as it is not required in this project, Try the CSS property `overflow-x`


##### Video of the horizontal flow

https://user-images.githubusercontent.com/58771507/179347033-3f8539cf-78e7-4b8b-8156-03554b4549f8.mp4